package.json

```diff
@@ -1,60 +1,81 @@
 {
-  "name": "adf-app",
-  "version": "0.0.1",
-  "license": "",
+  "name": "adf-cli-acs-aps-template",
+  "version": "2.6.0",
+  "blueprint": "Process and Content Services",
+  "description": "ACS APS ADF Application with Angular CLI",
+  "license": "MIT",
```

Comment:
Probably need Apache here as well.

Reply:
Thank you! Changed it in my last commit.